internal/k8s/ako_init.go

```diff
@@ -216,6 +216,74 @@ func (c *AviController) HandleConfigMap(k8sinfo K8sinformers, ctrlCh chan struct
 	return nil
 }
 
+func (c *AviController) AddBootupSecretEventHandler(k8sinfo K8sinformers, stopCh <-chan struct{}, startSyncCh chan struct{}) {
```

Comment:
Can we move this out to a different package? It looks like this will clutter the main AKO code base and this code is only used from the infra package?